cube/src/main/java/org/jboss/arquillian/ce/cube/CEEnvironmentProcessor.java

```diff
@@ -142,18 +144,24 @@ private void registerRoutes(CECubeConfiguration configuration, OpenShiftClient c
     }
 
     private void processTemplate(TestClass tc, OpenShiftAdapter client, CECubeConfiguration configuration)
-        throws DeploymentException {
+            throws DeploymentException {
         final StringResolver resolver = Strings.createStringResolver(configuration.getProperties());
         final Template template = ReflectionUtils.findAnnotation(tc.getJavaClass(), Template.class);
         final String templateURL = readTemplateUrl(template, configuration, resolver);
         final List<? extends OpenShiftResource> resources;
+
+        if (templateURL == null || templateURL.length() == 0) {
```

Comment:
@alesj I think it relies only upon who is writing it, if ce-arq doesn't find a template a warning message will be printed, so who is writing this test can use this message to know if a template is needed or not.